domain/test/main.test.js

```diff
@@ -110,8 +110,8 @@ describe('Тест для кладовки', function () {
         char.dmg = 0;
         char.critChance = 0;
         char.critDmg = 0;
-        let up = await klad.updateFullyInKladovka(db, coll, id, char);
-        assert(up === 1);
+        let repRes = await klad.replaceInKladovka(db, coll, id, char);
```

Comment:
All these tests for replace ... You always just assert the return value, but you don't assert that the actual action of a function was correct. Theoretically, your replace function can put some gibberish instead of original entity and all of your tests will still pass, so the practical value of such tests is not great. You should check that the item was indeed replaced (or not replaced, according to the test).

It is not that your approach is wrong per se, and actually, it is the opposite - it is the right way to do things according to the original unit test philosophy. But ... fck the original unit test philosophy.